3_Implementation/documentation/html/search/defines_3.js

```diff
@@ -0,0 +1,452 @@
+  ['test_5fassert_5fdouble_5fis_5fnot_5fneg_5finf_979',['TEST_ASSERT_DOUBLE_IS_NOT_NEG_INF',['../unity_8h.html#ade2d527619ba1c82fa8588a13f17099e',1,'unity.h']]],
+  ['test_5fassert_5fdouble_5fis_5fnot_5fneg_5finf_5fmessage_980',['TEST_ASSERT_DOUBLE_IS_NOT_NEG_INF_MESSAGE',['../unity_8h.html#aa9854cae981883e19b0959de5eda47b0',1,'unity.h']]],
+  ['test_5fassert_5fdouble_5fwithin_981',['TEST_ASSERT_DOUBLE_WITHIN',['../unity_8h.html#ab2392984572bbbab42898384470b8d06',1,'unity.h']]],
+  ['test_5fassert_5fdouble_5fwithin_5fmessage_982',['TEST_ASSERT_DOUBLE_WITHIN_MESSAGE',['../unity_8h.html#acdcdb54e0c13b85bc38ccd01be2ff1e5',1,'unity.h']]],
+  ['test_5fassert_5feach_5fequal_5fchar_983',['TEST_ASSERT_EACH_EQUAL_CHAR',['../unity_8h.html#a1fe65641ad53a4994f9ee50d9842d873',1,'unity.h']]],
+  ['test_5fassert_5feach_5fequal_5fchar_5fmessage_984',['TEST_ASSERT_EACH_EQUAL_CHAR_MESSAGE',['../unity_8h.html#a3cdeb9b48006bc10c0083e69d6627efd',1,'unity.h']]],
+  ['test_5fassert_5feach_5fequal_5fdouble_985',['TEST_ASSERT_EACH_EQUAL_DOUBLE',['../unity_8h.html#af753f9a9b11d53654e6355e558d0aa36',1,'unity.h']]],
+  ['test_5fassert_5feach_5fequal_5fdouble_5fmessage_986',['TEST_ASSERT_EACH_EQUAL_DOUBLE_MESSAGE',['../unity_8h.html#ab97080c7118d5da0f44e597b71b7241a',1,'unity.h']]],
+  ['test_5fassert_5feach_5fequal_5ffloat_987',['TEST_ASSERT_EACH_EQUAL_FLOAT',['../unity_8h.html#a94d6ea89d22cafe7cb422c01a94a5864',1,'unity.h']]],
+  ['test_5fassert_5feach_5fequal_5ffloat_5fmessage_988',['TEST_ASSERT_EACH_EQUAL_FLOAT_MESSAGE',['../unity_8h.html#a8d6790e2c5d098a89fb17d69b2197c38',1,'unity.h']]],
+  ['test_5fassert_5feach_5fequal_5fhex_989',['TEST_ASSERT_EACH_EQUAL_HEX',['../unity_8h.html#a96cdcca3abb32d076382836a081e151a',1,'unity.h']]],
+  ['test_5fassert_5feach_5fequal_5fhex16_990',['TEST_ASSERT_EACH_EQUAL_HEX16',['../unity_8h.html#a0c67423719eb669409dc3f2363134f68',1,'unity.h']]],
+  ['test_5fassert_5feach_5fequal_5fhex16_5fmessage_991',['TEST_ASSERT_EACH_EQUAL_HEX16_MESSAGE',['../unity_8h.html#a0d917ff51d3b6a4290e3a5c8a0036a63',1,'unity.h']]],
+  ['test_5fassert_5feach_5fequal_5fhex32_992',['TEST_ASSERT_EACH_EQUAL_HEX32',['../unity_8h.html#ac6e4a9d54c6777a4831f126f788ef3e0',1,'unity.h']]],
+  ['test_5fassert_5feach_5fequal_5fhex32_5fmessage_993',['TEST_ASSERT_EACH_EQUAL_HEX32_MESSAGE',['../unity_8h.html#a0df123e01453bc35292c065aa45623c1',1,'unity.h']]],
+  ['test_5fassert_5feach_5fequal_5fhex64_994',['TEST_ASSERT_EACH_EQUAL_HEX64',['../unity_8h.html#acd917a97a04a6a9b4dca540caba4961c',1,'unity.h']]],
+  ['test_5fassert_5feach_5fequal_5fhex64_5fmessage_995',['TEST_ASSERT_EACH_EQUAL_HEX64_MESSAGE',['../unity_8h.html#a8acf8b41b07a8928091257cb7850ae40',1,'unity.h']]],
+  ['test_5fassert_5feach_5fequal_5fhex8_996',['TEST_ASSERT_EACH_EQUAL_HEX8',['../unity_8h.html#a1464b59fbb630a2af105300dbc678157',1,'unity.h']]],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/2015pushkar/C-mini-Project-303459-LTTS/pullRequest?prid=7744834)
```suggestion
  ["test_5fassert_5feach_5fequal_5fhex8_996",['TEST_ASSERT_EACH_EQUAL_HEX8',['../unity_8h.html#a1464b59fbb630a2af105300dbc678157',1,'unity.h']]],
```